### ISSUE TITLE
SendAddonMessage disconnects player if zero-length message is sent

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#216]
The `C_ChatInfo.SendAddonMessage` API disconnects the sending player on Classic Era and Wrath Classic clients if a zero-length `message` parameter is supplied.

To reproduce, the following chat command can be executed.

```
/dump C_ChatInfo.SendAddonMessage("Test", "", "WHISPER", UnitName("player"))
```

On the retail/mainline client no disconnect occurs - the function returns `false` to indicate that the message wasn't queued for server transmission, thus the server never has a chance to disconnect the sender.

**Comment by funkydude** [comment]
Fixed in Wrath PTR build [3.4.0.45942](https://github.com/BigWigsMods/WoWUI/commit/d1b77c35ff084d853023f50e73d31be78907e7a1)

Now correctly returns `false`

**Comment by Meorawr** [comment]
I'll just close this now and assume Classic Era will pick up the changes in `$FUTURE_PATCH`.
